index.js

```diff
@@ -22,6 +22,32 @@ const validateClientId = clientId =>
 const validateRedirectUrl = redirectUrl =>
   invariant(typeof redirectUrl === 'string', 'Config error: redirectUrl must be a string');
 
+export const prefetchOnce = async ({
+  issuer,
+  redirectUrl,
+  clientId,
+  scopes,
+  serviceConfiguration,
+  dangerouslyAllowInsecureHttpRequests = false,
+}) => {
+  if (Platform.OS === 'android') {
+    validateIssuerOrServiceConfigurationEndpoints(issuer, serviceConfiguration);
+    validateClientId(clientId);
+    validateRedirectUrl(redirectUrl);
+
+    const nativeMethodArguments = [
+      issuer,
+      redirectUrl,
+      clientId,
+      scopes,
+      serviceConfiguration,
+      dangerouslyAllowInsecureHttpRequests,
+    ];
+
```

Comment:
Idk, I'm relatively new to JS and promises, but we should return/await here but it gives linting error. Is it maybe because of the eslint version?

I'd be happy to change it if I could just find out to what... :(

Reply:
No need do `await` here, because you're returning a function that is `await`ed when it's called